src/components/backgroundselector/BackgroundSelectorDirective.js

```diff
@@ -78,6 +74,29 @@ goog.require('ga_topic_service');
             });
             return isValid;
           };
+          var updateDefaultBgOrder = function(bgLayers) {
+            bgLayers = bgLayers ? bgLayers : [];
+            defaultBgOrder = [];
+            bgLayers.forEach(function(bgLayerId) {
+              if (bgLayerId === voidLayer.id) {
+                defaultBgOrder.push(voidLayer);
+              } else {
+                defaultBgOrder.push({
+                  id: bgLayerId,
+                  label: gaLayers.getLayerProperty(bgLayerId, 'label')
```

Comment:
Yes, that's correct. We have specific translations (front-end only) for the background layers which don't correspond to the translations from the backend. We need to keep this functionality.